source/vistas/core/color.py

```diff
@@ -117,7 +123,7 @@ def hsva_list(self):
 
 
 def interpolate_color(range, min_color: Color, max_color: Color, t):
-    """ Linearly interpolate in the HSV color space """
+    """ Linearly interpolate in the HSV color space. Copies colors in RGB space to HSV. """
```

Comment:
I elected for 'copies' over 'converts' to make it clear that the values based in were not being overwritten. That said, I'm okay with 'converts'.